TLIndexPathTools/Data Model/TLIndexPathUpdates.m

```diff
@@ -403,6 +403,7 @@ - (void)performBatchUpdatesOnCollectionView:(UICollectionView *)collectionView c
 - (void)setModifiedItems:(NSArray *)modifiedItems
 {
     _modifiedItems = [modifiedItems copy];
+    _hasChanges = _modifiedItems.count != 0;
```

Comment:
Can you change this to:

    self.hasChanges = self.hasChanges || _modifiedItems.count != 0;